python/lsst/dax/imgserv/templates/uws_job_result.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<uws:job xmlns:uws="http://www.ivoa.net/xml/UWS/v1.0">
+  <uws:jobId>{{ uws_job_id }}</uws:jobId>
+  <uws:runId>test</uws:runId>
+  <uws:ownerId xsi:nil="true" />
+  <uws:phase>{{ soda_job_phase }}</uws:phase>
+  <uws:startTime/>
+  <uws:endTime/>
+  <uws:executionDuration>{{ soda_exec_duration }}</uws:executionDuration>
+  <uws:parameters>
+    <uws:parameter id="LANG">SODA</uws:parameter>
+    <uws:parameter id="REQUEST">getImage</uws:parameter>
```

Comment:
This one is getImage, but the other in uws_job_descriptor is get_image?

These two files look very similar.  Maybe we can get this down to the same one file?

Reply:
Updated both to 'get_image_async', which is the latest name of the API.

Reply:
does it need to be the name of the function?  maybe just "cutout" would be good since that's what this is doing?

Reply:
"cutout" - it is, as it's in the response.  The API name as is won't be useful in the future anyway, once sync is done on top of async.